index.js

```diff
@@ -180,3 +180,5 @@ function getReqUrl (source, cache, base, opts) {
 
   return url
 }
+
+module.exports = fastProxy
```

Comment:
I means we should contain all of them, otherwise `TypeScript` use `export default` will not match.
```suggestion
module.exports = fastProxy
module.exports.default = fastProxy
module.exports.fastProxy = fastProxy
```

Reply:
I can see you are using this pattern for other fastify projects as well. I have no problem to support it if this is a guideline...

Reply:
The main reason for this pattern is to match the types and different community use-case.
In TypeScript. 
`export default` should match with `module.exports.default = `.
`export = ` match with `module.export = `.

Since, we should not breaks any users. It is just better to support all three of them or update the type to `export =`.